docs/advanced/custom_rest_handler.md

```diff
@@ -111,17 +109,17 @@ if __name__ == "__main__":
 
 > UCC 5.18.0 natively supports custom REST handlers for the modular inputs.
 
-One of the common scenarios is to delete a checkpoint after you are deleting an
-input in the Inputs page. Otherwise, users may face the wierd consequences if 
-they create an input with the same name as the input that was deleted and this 
-newly created input will be reusing the old checkpoint because the names of 
+One common scenario is to delete a checkpoint after you delete an
+input in the Inputs page. Otherwise, users may face strange consequences if 
+they create an input with the same name as the input that was deleted, and this 
+newly created input will reuse the old checkpoint, because the names of 
 the inputs are the same. We would like to avoid this situation in the add-on.
 
-This can be done without a need to modify the REST handler code generated 
+This can be done without a need to modify the REST handler code
 automatically by running `ucc-gen`.
```

Comment:
yea, i had problems understanding that one too, but i think the 'base' sentence was something like this 
```This can be done automatically by running `ucc-gen`.```
and then the rest was added 
but maybe it the end something like this might be the most understandable
```This can be done automatically by running `ucc-gen`, without a need to modify the REST handler code```